diff --git a/arch/arm/boot/dts/st/stm32mp135f-dhcor-dhsbc.dts b/arch/arm/boot/dts/st/stm32mp135f-dhcor-dhsbc.dts
index 40943ae51435..63898f5e99e4 100644
--- a/arch/arm/boot/dts/st/stm32mp135f-dhcor-dhsbc.dts
+++ b/arch/arm/boot/dts/st/stm32mp135f-dhcor-dhsbc.dts
@@ -76,28 +76,23 @@ channel@12 {
 
 &ethernet1 {
 	nvmem-cell-names = "mac-address";
 	nvmem-cells = <&ethernet_mac1_address>;
 	phy-handle = <&ethphy1>;
-	phy-mode = "rgmii-id";
+	phy-mode = "rmii";
 	pinctrl-0 = <&eth1_rgmii_pins_a>;
 	pinctrl-1 = <&eth1_rgmii_sleep_pins_a>;
 	pinctrl-names = "default", "sleep";
-	st,ext-phyclk;
 	status = "okay";
 
 	mdio {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "snps,dwmac-mdio";
 
 		ethphy1: ethernet-phy@1 {
-			/* RTL8211F */
-			compatible = "ethernet-phy-id001c.c916";
-			interrupt-parent = <&gpiog>;
-			interrupts = <12 IRQ_TYPE_LEVEL_LOW>;
-			reg = <1>;
+			reg = <0>;
 			realtek,clkout-disable;
 			realtek,ssc-enable;
 			reset-assert-us = <15000>;
 			reset-deassert-us = <55000>;
 			reset-gpios = <&gpioa 11 GPIO_ACTIVE_LOW>;
